### DISCUSSION TITLE
Would it be possible to also implement PulID?

### DISCUSSION THREAD

**Discussion by azrahello** [filipstrand/mflux#547] [Ideas & Suggestions: Features]
I saw that they implemented it this way in DrawThings, so I was wondering if it could also work with MLX
Always compliments for this amazing project… which has now become indispensable :P

**Comment by filipstrand** [comment]
@azrahello Thank you very much :) 

Had not seen this before. Looks really cool and a good fit for the project. Will put this on the todo list.
